src/main/java/io/phasetwo/keycloak/magic/MagicLink.java

```diff
@@ -119,7 +120,8 @@ public static MagicLinkActionToken createActionToken(
       String scope,
       String nonce,
       String state,
-      Boolean rememberMe) {
+      Boolean rememberMe,
```

Comment:
ditto above
> Since this method is public, it should not be altered. Make a new method with the new arguments, and update this method to call the new one with the the default (true).